tests/Makefile

```diff
@@ -301,6 +301,7 @@ test-lz4c: lz4c datagen
 	./datagen -g256MB | $(LZ4)c -l -v    | $(LZ4)c   -t
 
 test-lz4c32: CFLAGS+=-m32
+test-lz4c32: MOREFLAGS+="-I/usr/include/x86_64-linux-gnu"
```

Comment:
I'm not sure if it's needed here anymore because I've started using in Circle.yml:
`make -C tests test32 MOREFLAGS="-I/usr/include/x86_64-linux-gnu" && make clean`